src/components/Steps/Map/index.js

```diff
@@ -6,16 +6,19 @@ import PropTypes from 'prop-types';
 
 import Area from './Area';
 import Station from './Station';
+import Slidedown from './Slidedown';
+import CITIES from './../../../const/cities';
 
-const RADIUS = 4;
-const DEFAULT_ZOOM = 11;
+const RADIUS = 200;
+const DEFAULT_ZOOM = 10;
 
 class MapStep extends Component {
     constructor(props) {
         super(props);
         this.state = {
             stations: [],
             streetsInAreas: [],
+            closed: [],
```

Comment:
Again isn't it better to hold only index of active one ? If I undestand it right, we have always 1 open, right ?